README.md

```diff
@@ -31,9 +31,9 @@ the top right of the page.
 
 Issues and PRs towards the documentation should be filed in the [website repo](https://github.com/cert-manager/website/).
 
-For the common use-case of automatically issuing TLS certificates to
+For the common use case of automatically issuing TLS certificates to
 Ingress resources, aka a [kube-lego](https://github.com/jetstack/kube-lego)
-replacement, see the [cert-manager nginx ingress quick start
+replacement, see the [cert-manager Nginx ingress quick start
```

Comment:
I don't think this needs changing. I only see nginx written as nginx or NGINX in other places.

```suggestion
replacement, see the [cert-manager nginx ingress quick start
```